examples/python-code/README.md

```diff
@@ -0,0 +1,9 @@
+# Rhino Health Examples - Generalized Compute
+This folder contains examples for using Rhino Health's Python Code Object capability
+
+## Table of Contents
+- [data-quality](./dcm2png/README.md) - Process input CSV files that are in the OMOP Common Data Model, which is the intermediary format created by the Rhino CoPilot and used by Medinexo sites to represent their data. 
```

Comment:
The link here points to `dcm2png` which isn't a folder that exists under `examples/python-code`. Also - since Python Code is just a type of GC, users can take the code from the generalized-compute examples folder and use that even with a "Python Code" object if they pass in the right requirements, etc.